dashboard/test/ui/features/teacher-homepage.feature

```diff
@@ -12,3 +12,13 @@ Feature: Using the teacher homepage sections feature
     # Create my second section (via the button in OwnedSections)
     When I create a new section
     Then the section table should have 2 rows
+
+  Scenario: Loading teacher homepage with course experiment enabled
+    Given I enable the "subgoals-group-a" course experiment
```

Comment:
I'm guessing we don't need to disable the experiment at the end of our test because our Scenarios should run in isolation (except on mobile?) and this is the last scenario in this feature.  Just confirming that's your understanding too.

Reply:
Experiments in this case are per user, not per browser, so the experiment will effectively go away with the user at the end of each scenario, mobile or not.